Tests/Editor/TextureTests.cs

```diff
@@ -0,0 +1,58 @@
+using NUnit.Framework;
+using Unity.ProjectAuditor.Editor;
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.ProjectAuditor.EditorTests
+{
+    class TextureTests : TestFixtureBase
+    {
+        const int resolution = 1;
```

Comment:
please use `k_ `and `m_` prefixes as appropriate

Reply:
Updated TextureTests.cs with k_ and m_